src/Utils/ObjectMixin.php

```diff
@@ -317,6 +306,45 @@ public static function getMagicMethods($class)
 
 
 	/**
+	 * Returns array of magic properties defined by annotation @property.
+	 * @return array
+	 */
+	public static function getMagicProperties($class)
```

Comment:
But on how this method is actually used, having `getMagicProperty($class, $name)` (likely returning either NULL or 0 if such property does not exist) may be reasonable.